charts/pelorus/templates/dashboard-snapshot-cronjob.yaml

```diff
@@ -38,7 +38,14 @@ subjects:
   namespace: {{ .Release.Namespace }}
 
 ---
+{{- /*
+Issue #386 - we need to use different APIs
+*/}}
+{{ if semverCompare ">=1.21" .Capabilities.KubeVersion.Version -}}
 apiVersion: batch/v1
+{{- else }}
+apiVersion: batch/v1beta1
+{{- end }}
 kind: CronJob
```

Comment:
nice! didn't realize jinja was an option :)  that's good to know

Reply:
@weshayutin yeah, but there are quite a lot of limitations... first was playing with:

{{- $useCronJobV1 := .Capabilities.APIVersions.Has "batch/v1" -}}

Even if it should work... Helm has some wired problems while testing and decided to check for the k8s version in which API was changed.

Reply:
That should be "batch/v1/CronJob", no? Maybe that's the issue?

Reply:
@KevinMGranger yes! That worked perfectly. Updated PR with your suggestion.